docker-compose.yml

```diff
@@ -12,7 +12,8 @@ services:
           MYSQL_USER: "${MYSQL_USER}"
 
     pathfinder:
-        image: kryptedgaming/pathfinder:latest
+        image: pathfinder:latest
```

Comment:
Not accepting changes to `docker-compose.yml`, make sure you keep your local version uncommited

Reply:
yep, sorry that was debug stuffs left over.